src/main/java/online/kingdomkeys/kingdomkeys/client/gui/synthesis/SynthesisCreateScreen.java

```diff
@@ -221,7 +221,7 @@ public void render(@NotNull GuiGraphics gui, int mouseX, int mouseY, float parti
 		matrixStack.pushPose();
 		{
 			matrixStack.translate(width * 0.03F + 45, (height * 0.15) - 18, 1);
-			gui.drawString(minecraft.font, Utils.translateToLocal("Page: " + (page + 1)), 0, 10, 0xFF9900);
+			gui.drawString(minecraft.font, Gui_Shop_Page + ": " + (page + 1)), 0, 10, 0xFF9900);
```

Comment:
Strings.Gui_Shop_Page
Could include the colon and page number in the translation